broker-core/src/test/java/io/zeebe/broker/workflow/TimerStartEventTest.java

```diff
@@ -78,17 +81,16 @@
 
   private PartitionTestClient testClient;
 
-  static {
-    ProcessBuilder builder = Bpmn.createExecutableProcess("process_4");
-    builder.startEvent("start_4").timerWithCycle("R/PT2S").endEvent("end_4");
-    multiStartModel =
-        builder.startEvent("start_5").timerWithCycle("R/PT3S").endEvent("end_5").done();
-
-    builder = Bpmn.createExecutableProcess("process_5");
+  private static BpmnModelInstance createTimerAndMessageStartEventsModel() {
```

Comment:
Hmm, I'm actually not sure if the `bpmn-model` module is meant to be used externally by users or primarily for us, so not sure if it's so important that it can do multiple start events? I see it as useful but perhaps not pressing. @menski ?